readthedocs/settings/base.py

```diff
@@ -172,7 +172,14 @@ def USE_PROMOS(self):  # noqa
     MEDIA_ROOT = os.path.join(SITE_ROOT, 'media/')
     MEDIA_URL = '/media/'
     ADMIN_MEDIA_PREFIX = '/media/admin/'
-    STATICFILES_DIRS = [os.path.join(SITE_ROOT, 'readthedocs', 'static')]
+    STATICFILES_DIRS = [
+        os.path.join(SITE_ROOT, 'readthedocs', 'static'),
+        os.path.join(SITE_ROOT, 'media'),
```

Comment:
This still seems a bit weird to me to collect files from a path (`/media`) and put them in a subdirectory (`/media/static`) but I guess it's ok with the `SelectiveFileSystemFinder`.

Reply:
Yeah agreed. I only worry that there are some deep parts requiring the media path, as this pattern goes back to early django I believe. If we run into any weirdness here, I think we can probably justify taking on work to separate static.